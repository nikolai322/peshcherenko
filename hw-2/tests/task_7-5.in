111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq
