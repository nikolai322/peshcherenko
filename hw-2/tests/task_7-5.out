There is too much of input data
